content/project/governance.adoc

```diff
@@ -282,7 +282,7 @@ Every weekend a new release is built from the master branch and released, in var
 
 === LTS Releases
 
-Every three months or so we pick a prior release as the new long-term support (LTS) release and then create the ‘stable’ branch, from that release point. This branch gets important bug fixes backported from the master branch, and further patch releases are built roughly every two weeks until the next LTS baseline is chosen. See link:https://wiki.jenkins.io/JENKINS/LTS+Release+Line[LTS Release Line] for more details.
+Every three months or so we pick a prior release as the new long-term support (LTS) release and then create the ‘stable’ branch, from that release point. This branch gets important bug fixes backported from the master branch, and further patch releases are built roughly every two weeks until the next LTS baseline is chosen. See link:https://www.jenkins.io/download/lts/[LTS Release Line] for more details.
```

Comment:
Can use a local reference rather than the full URL, I believe
```suggestion
Every three months or so we pick a prior release as the new long-term support (LTS) release and then create the ‘stable’ branch, from that release point. This branch gets important bug fixes backported from the master branch, and further patch releases are built roughly every two weeks until the next LTS baseline is chosen. See link:/download/lts/[LTS Release Line] for more details.
```